### PR TITLE
added security data copy script

### PR DESCRIPTION
**Pull request by gabialverga**
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: scripts/security_data_copy.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from distutils.dir_util import copy_tree
+from argparse import ArgumentParser
+
+parser = ArgumentParser()
+parser.add_argument('--source_path', type=str)
+parser.add_argument('--security_destination', type=str)
+parser.add_argument('--experiment', type=str)
+
+destination_path_name = '{}/experiment_{}'
```
Comment (dfquaresma) [inline scripts/security_data_copy.py]: Could you name the params in this kind of string format? 
For example:
```suggestion
destination_path_name = '{destination}/experiment_{experiment}'
```
And then, when using this string, you could do:
```
destination_path_name.format(destination=security_destination, experiment=experiment)
```

I think it improves the readability and put this script into the pattern in which others python script we use are in.

File: scripts/security_data_copy.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+args = parser.parse_args()
+source_path = args.source_path
+security_destination = args.security_destination
+experiment = args.experiment
+destination_path = destination_path_name.format(security_destination, experiment)
+
+copy_tree(source_path, destination_path)
```
Comment (dfquaresma) [inline scripts/security_data_copy.py]: Please, could you add an empty line below here? [Link to further reading](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

```suggestion
copy_tree(source_path, destination_path)

```

**Comment by thiagomanel** [comment]
Hi @gabialverga and others: I have a feeling this code could be simpler. Maybe, it could be replaced by a `cp` or `scp` call at a higher level on the workflow. As I'm not aware of the entire stack, instead of making a request for change here, I'd let it as general advice: bash is a time saver  (for sure, for more complex stuff python is a way to go).

**Comment by dfquaresma** [comment]
> Hi @gabialverga and others: I have a feeling this code could be simpler. Maybe, it could be replaced by a `cp` or `scp` call at a higher level on the workflow. As I'm not aware of the entire stack, instead of making a request for change here, I'd let it as general advice: bash is a time saver (for sure, for more complex stuff python is a way to go).

I agree. I think we can stay using the shell to cp and SCP. 

@gabialverga, since we are working together these weeks and we are not using that script, feel free to close this PR. We were talking about improving it to do SCP, but it would cost time and the SCP commands we use to know is not costing anything to us.

**Pull request closed without merging**